skl2onnx/algebra/onnx_operator.py

```diff
@@ -6,7 +6,7 @@
 import numpy as np
 from scipy.sparse import coo_matrix
 from ..proto import TensorProto
-from ..common._topology import Variable, Scope
+from ..common._topology import Variable, Scope, _update_domain_version
```

Comment:
No PEP warning about importing a private func?

Reply:
I guess not, private Inside the module, users should not use it.